Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -453,10 +453,21 @@ theorem toReal_eq_toReal_iff' {x y : ℝ≥0∞} (hx : x ≠ ⊤) (hy : y ≠ 
 theorem one_lt_two : (1 : ℝ≥0∞) < 2 := Nat.one_lt_ofNat
 #align ennreal.one_lt_two ENNReal.one_lt_two
 
-@[simp] theorem two_ne_top : (2 : ℝ≥0∞) ≠ ∞ := coe_ne_top
+@[simp] theorem ofNat_ne_top {n : ℕ} [n.AtLeastTwo] : (no_index (OfNat.ofNat n : ℝ≥0∞)) ≠ ∞ :=
```

Comment:
Please copy the standard "see note" that appears before every `OfNat.ofNat` lemma in mathlib.